docs/add-annotation-type.md

```diff
@@ -0,0 +1,27 @@
+# Adding support for new Annotation types
+See [Image Point Annotations](https://github.com/box/box-annotations/tree/master/src/image) for a basic implementation of an annotation type.
+
+### The following files/abstract methods need to be implemented:
+* controllers/{**TYPE**}Controller.js (extends [AnnotationModeController](https://github.com/box/box-annotations/blob/master/src/controllers/AnnotationModeController.js)): This class is the communicator between the Annotator and the Annotation threads. Each controller maintains an object of pages which contains a [red/black tree](https://github.com/mourner/rbush)
```

Comment:
Reword this not not use specific terms like "Annotator".

Reply:
I explained what these terms are before diving into what needs to be changed.